backend/.env.bak

```
@@ -0,0 +1,164 @@
+# Maximum concurrent wiki generations
+WIKI_MAX_CONCURRENT_GENERATIONS=5
+# Background polling interval (seconds)
+WIKI_RESULT_POLL_INTERVAL_SECONDS=30
+# Background polling batch size
+WIKI_RESULT_POLL_BATCH_SIZE=20
+# Maximum content size (bytes, default 10MB)
+WIKI_MAX_CONTENT_SIZE=10485760
+# Base URL for internal wiki content writer
+WIKI_CONTENT_WRITE_BASE_URL=http://backend:8000
+
+CHAT_MCP_ENABLED=True
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The CHAT_MCP_ENABLED key is duplicated

(DuplicatedKey)